modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -82,6 +103,158 @@ export async function createCombinedKey(
   };
 }
 
+/**
+ * Creates the SignShare with User XShare and YShare Corresponding to BitGo
+ * @param {XShare} xShare User secret xShare
+ * @param {YShare} yShare YShare from Bitgo
+ * @returns {Promise<SignShare>}
+ */
+export async function createUserSignShare(xShare: XShare, yShare: YShare): Promise<SignShare> {
+  if (xShare.i === ShareKeyPosition.BITGO) {
+    throw new Error(`Invalid XShare, XShare doesn't belong to the User`);
+  }
+
+  if (yShare.i === ShareKeyPosition.BITGO || yShare.j !== ShareKeyPosition.BITGO) {
+    throw new Error('Invalid YShare provided for sign');
+  }
+  return MPC.signShare(xShare, yShare);
+}
+
+/**
+ * Creates the Gamma Share with User WShare and AShare From BitGo
```

Comment:
nit: Createes the Gamma Share and Mu Share with User WShare and AShare from BitGo